app/ai/voice/agents/breeze_buddy/stt/__init__.py

```
@@ -60,62 +230,175 @@ async def get_stt_service(
+                    fallback = _build_deepgram_fallback(vad_events=False)
+                    logger.info(
+                        "Probe call: Soniox primary + Deepgram fallback "
+                        "(ServiceSwitcher, vad_events=False)"
+                    )
+                    return STTServiceResult(
+                        service=stt_service,
+                        provider="soniox",
+                        fallback_service=fallback,
+                        is_probe_call=True,
+                    )
+                except Exception as probe_err:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)